frontend/src/components/MeetingRoom/ArchivingButton/ArchivingButton.tsx

```diff
@@ -52,7 +57,9 @@ const ArchivingButton = ({ handleClick }: ArchivingButtonProps): ReactElement =>
 
   const handleClose = () => {
     setIsModalOpen(false);
-    if (isSmallViewport && handleClick) {
+
+    // If the ArchivingButton is in the ToolbarOverflowMenu, we close the modal and the menu
+    if (shownButtons < 4 && handleClick) {
```

Comment:
doesn't the overflowButton prop tell us this? 
Let's try to avoid fragile magic numbers like this please 🙏